packages/xstate-react/src/useConstant.ts

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+
+type ResultBox<T> = { v: T };
+
+export default function useConstant<T>(fn: () => T): T {
```

Comment:
this just a copy of a little helper I'm using (and which you kinda had inlined here:
https://github.com/davidkpiano/xstate/blob/8545f851e0b1ef9c374e1845abb9c3a0f7735e80/packages/xstate-react/src/index.ts#L67-L70

I'm just using it a way to hide this dance with a ref and also make sure that a defined TS type is received at a call site (null can't pop up when trying to consume content of this).

I have this packaged in an npm package, but I know you don't quite like external deps 😉 

Also - can just inline this if you think this is over complicated and unnecessary here.

Reply:
This is fine as-is